### ISSUE TITLE
background images

### ISSUE THREAD

**Issue by rimmesbe** [michaeljbaker22/Tea-Cozy#3]
The background image of the location section is repeating itself since it can't cover the width. You can stop this from happening and center the background image:
background: url (...) no-repeat center center;

https://github.com/michaeljbaker22/Tea-Cozy/blob/master/resources/style.css#L96

**Comment by michaeljbaker22** [comment]
yep! I just realized this when I was going over it this morning and saw this style control in the next unit, oops!
